# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

BatteryActionGoal action_goal
BatteryActionResult action_result
BatteryActionFeedback action_feedback
